docs/pipelines/core/matcher.md

```diff
@@ -43,14 +43,9 @@ This snippet is complete, and should run as is.
 
 The pipeline can be configured using the following parameters :
 
-| Parameter             | Explanation                                                    | Default                 |
-| --------------------- | -------------------------------------------------------------- | ----------------------- |
-| `terms`               | Terms patterns. Expects a dictionary.                          | `None` (use regex only) |
-| `regex`               | RegExp patterns. Expects a dictionary.                         | `None` (use terms only) |
-| `term_matcher`        | Which algorithm should we use : `exact` or `simstring`         | `"exact"`               |
-| `term_matcher_config` | Config of the algorithm (`SimstringMatcher`'s for `simstring`) | `{}`                    |
-| `attr`                | spaCy attribute to match on (eg `NORM`, `LOWER`)               | `"TEXT"`                |
-| `ignore_excluded`     | Whether to skip excluded tokens during matching                | `False`                 |
+::: edsnlp.pipelines.core.matcher.factory.create_component
+    options:
+        only_parameters: true
```

Comment:
It is not explicative

Reply:
this will be once https://github.com/aphp/edsnlp/pull/196 is merged :)